src/monster_minotaur.cpp

```diff
@@ -834,6 +834,34 @@ void actMinotaurCeilingBuster(Entity* my)
 								list_RemoveNode(entity->mynode);
 							}
 						}
+						else if (	entity->behavior == &actStalagCeiling	||
+									entity->behavior == &actStalagFloor		||
+									entity->behavior == &actStalagColumn 
+								)
+						{
+							// spawn several rock items
+							int c, i = rand() % 4;
+							for ( c = 0; c < i; ++c )
+							{
+								Entity* childEntity = spawnGib(my);
```

Comment:
This should differentiate between the client and the server. You may recall the ceiling buster crash in multiplayer from a bit back.